src/main/resources/locale.properties

```diff
@@ -373,3 +373,4 @@ words.refMessage=
 words.reputation=
 words.reputationOffset=
 words.unknown=
+words.role=
```

Comment:
You sort the keys in the release PR again?

Reply:
Yes